renderdoc/driver/gl/gl_resources.cpp

```diff
@@ -543,7 +547,7 @@ GLenum GetSizedFormat(const GLHookSet &gl, GLenum target, GLenum internalFormat,
   }
 
   GLint red, depth, stencil;
-  if(HasExt[ARB_internalformat_query2] && gl.glGetInternalformativ)
+  if(gl.glGetInternalformativ)
```

Comment:
Ah, I ran into the exact opposite problem where glGetInternalformativ is present and returned something that looks correct, but the extension ARB_internalformat_query2 failed the check. I'll revert this one and take a deeper look at what is happening.